### PR TITLE
Add BrowserWindow.prototype.setThumbnailToolTip

### PR DESCRIPTION
**Pull request by miniak**
https://msdn.microsoft.com/en-us/library/windows/desktop/dd391702(v=vs.85).aspx

### CODE REVIEW

File: atom/browser/api/atom_api_window.cc [hunk 1]
```diff
@@ -858,6 +864,7 @@ void Window::BuildPrototype(v8::Isolate* isolate,
       .SetMethod("unhookWindowMessage", &Window::UnhookWindowMessage)
       .SetMethod("unhookAllWindowMessages", &Window::UnhookAllWindowMessages)
       .SetMethod("setThumbnailClip", &Window::SetThumbnailClip)
+      .SetMethod("setThumbnailTooltip", &Window::SetThumbnailTooltip)
```
Comment (kevinsawicki) [inline atom/browser/api/atom_api_window.cc]: The method on `Tray` is called [`setToolTip`](http://electron.atom.io/docs/all/#traysettooltiptooltip) so I wonder if this should be `setThumbnailToolTip` for consistency.

Comment (miniak) [inline atom/browser/api/atom_api_window.cc]: `setToolTip` sounds too generic, suggesting that it affect the window itself. It's a companion method to `setThumbnailClip`.

Comment (MarshallOfSound) [inline atom/browser/api/atom_api_window.cc]: @kevinsawicki Was referring to capitalizing (the second T in ToolTip) `"setThumbnailToolTip"`

Comment (miniak) [inline atom/browser/api/atom_api_window.cc]: I guess it's too late... I haven't read his comment properly :)
Renamed to `setThumbnailToolTip`, even though I don't like it much, as I think tooltip is a single word.

Comment (miniak) [inline atom/browser/api/atom_api_window.cc]: I've also just noticed that the branch name is incorrect :)

Comment (MarshallOfSound) [inline atom/browser/api/atom_api_window.cc]: @miniak As is the MSDN link at the top of the PR, but what's a link and a branch name between friends 😆

File: atom/browser/api/atom_api_window.cc [hunk 2]
```diff
@@ -667,6 +667,12 @@ bool Window::SetThumbnailClip(const gfx::Rect& region) {
   return window->taskbar_host().SetThumbnailClip(
       window_->GetAcceleratedWidget(), region);
 }
+
+bool Window::SetThumbnailToolTip(const std::string& tooltip) {
```
Comment (zcbenz) [inline atom/browser/api/atom_api_window.cc]: We can pass `const base::string16&` directly here, which is more efficient than converting `std::string` to UTF16 later.

File: atom/browser/ui/win/taskbar_host.h [hunk 3]
```diff
@@ -42,7 +42,10 @@ class TaskbarHost {
       HWND window, const gfx::Image& overlay, const std::string& text);
 
   // Set the region of the window to show as a thumbnail in taskbar.
-  bool TaskbarHost::SetThumbnailClip(HWND window, const gfx::Rect& region);
+  bool SetThumbnailClip(HWND window, const gfx::Rect& region);
+
+  // Set the tooltip for the thumbnail in taskbar.
+  bool setThumbnailToolTip(HWND window, const std::string& tooltip);
```
Comment (zcbenz) [inline atom/browser/ui/win/taskbar_host.h]: `set` => `Set`.

Comment (miniak) [inline atom/browser/ui/win/taskbar_host.h]: That's what happens at 3:00 AM being tired :)

File: atom/browser/api/atom_api_window.cc (hunk 2 above)
Comment (zcbenz) [inline atom/browser/api/atom_api_window.cc]: :+1:

**Pull request merged**